src/lsm/lsm_manager.c

```diff
@@ -469,7 +429,8 @@ __lsm_manager_run_server(WT_SESSION_IMPL *session)
 				    session, WT_LSM_WORK_FLUSH, 0, lsm_tree));
 				WT_ERR(__wt_lsm_manager_push_entry(
 				    session, WT_LSM_WORK_BLOOM, 0, lsm_tree));
-				WT_ERR(__wt_verbose(session, WT_VERB_LSM,
+				WT_ERR(__wt_verbose(session,
+				    WT_VERB_LSM_MANAGER,
```

Comment:
I think having a separate category is a good idea.  There are probably more verbose messages that would be useful in this category.  Reconfig to start/stop threads and clearing all work units from a particular tree are two areas that come to mind.